packages/amplify-e2e-tests/src/__tests__/api_10.test.ts

```diff
@@ -125,10 +127,10 @@ describe('amplify add api (GraphQL)', () => {
     expect(beforeAdminConfig).toMatchSnapshot();
 
     const customRolesConfig = {
-      adminRoleNames: ['myAdminRoleName'],
+      adminRoleNames: ['myAdminRoleName', 'myEnvFunction-${env}'],
     };
     setCustomRolesConfig(projRoot, name, customRolesConfig);
-    await apiGqlCompile(projRoot);
+    await apiGqlCompile(projRoot, true);
```

Comment:
Why is it necessary to use true here instead of the default false?

Reply:
The graphql compiler seems to use what is installed in node_modules unless true is set and then it uses the compiler which includes the code I updated.

Reply:
The compiler is resolved here: https://github.com/aws-amplify/amplify-category-api/blob/c79232ff8a2f7809447f51233a2dbe0046c25e47/packages/amplify-e2e-core/src/categories/api.ts#L25

And it uses either the internal or pre installed CLI determined off the boolean flag. 

https://github.com/aws-amplify/amplify-category-api/blob/c79232ff8a2f7809447f51233a2dbe0046c25e47/packages/amplify-e2e-core/src/index.ts#L30-L35